js/windowMgr.js

```diff
@@ -319,7 +316,6 @@ function doCreateMainWindow(initialUrl, initialBounds) {
         }
     });
 
```

Comment:
As we have implemented `electron-spellchecker` module which also takes care of the contextMenu, so removing `electron-context-menu` from dependency